src/index.html

```diff
@@ -11,7 +11,184 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+
+        <div class="input-block">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+          >
+        </div>
+
+        <div class="input-block">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+          >
+        </div>
+
+        <div class="input-block">
+          <label for="years">How old are you?</label>
+          <input
+            type="number"
+            name="years"
+            id="years"
+            value="12"
+            max="100"
+            min="12"
+          >
+        </div>
+
+        <div class="input-block">
+          <label for="birth">Full date of birth:</label>
+          <input
+            type="date"
+            name="birth"
+            id="birth"
+          >
+        </div>
+
+        <div>
+          <label for="term">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="term"
+            id="term"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="field">
+        <legend>Registration</legend>
+
+        <div class="input-block">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+        <div>
+          <label for="pswd">Password:</label>
```

Comment:
don't cut words like that
```suggestion
          <label for="password">Password:</label>
```